java/spacewalk-java.spec

```diff
@@ -403,6 +402,13 @@ done
 %endif
 
 %build
+PRODUCT_NAME="SUSE Manager"
+%if 0%{?sle_version} && !0%{?is_opensuse}
+PRODUCT_NAME="Beta"
```

Comment:
Why "Beta" ? With this the product name will never be "SUSE Manager" because you overwrite it in all cases.

Reply:
I copied that from another place in the spec file... then that other one needs rethinking too ;)

Reply:
The other place is where we set the "tag". AFAIK there is a comment to remove this when Beta time is over. At that place we need to differ between "Beta" and "4.1".
Here not.

Reply:
fixed